index.js

```diff
@@ -4,20 +4,43 @@
 var fs   = require('fs');
 var path = require('path');
 var Funnel = require('broccoli-funnel');
+var Merge = require('broccoli-merge-trees');
+var existsSync = require('exists-sync');
+var fastbootTransform = require('fastboot-transform');
 
 module.exports = {
   name: 'ember-cli-photoswipe',
 
   included: function(app) {
     this.app  = app;
     var psDir = app.bowerDirectory + '/photoswipe';
+    app.import(psDir + '/dist/photoswipe.css');
+    app.import(psDir + '/dist/default-skin/default-skin.css');
 
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      app.import(psDir + '/dist/photoswipe.css');
-      app.import(psDir + '/dist/default-skin/default-skin.css');
-      app.import(psDir + '/dist/photoswipe.js');
-      app.import(psDir + '/dist/photoswipe-ui-default.min.js');
+    console.log(psDir);
```

Comment:
This console.log probably should be removed before merging.

Reply:
@kratiahuja Seems a pitty this has gone no further? :)